model/Build/Classes/Build.md

```diff
@@ -57,3 +58,15 @@ Note that buildStart and buildEnd are optional, and may be omitted to simplify c
 - environment
   - type: /Core/DictionaryEntry
   - minCount: 0
+
+## External LifecycleScopedRelationship Constraints
+
+- relationshipType: inputOf OR buildTool OR configOf
+  - toType: /Build/Build
```

Comment:
In the discussion on the relationship type restrictions, we decided to only have a fromType sub-bullet since the toType is typically restricted by the relationshipType across all profiles.  I'm thinking the from type would be `/Core/Artifact`.